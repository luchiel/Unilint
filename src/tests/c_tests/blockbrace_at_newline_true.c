struct a
{
    int b;
} c;

int main()
{
    for(int i = 0; i < 100; ++i) {
        //do nothing
    }
    return 1;
}

