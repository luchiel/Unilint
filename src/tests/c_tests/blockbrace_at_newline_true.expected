8:34: { is not at new line
